src/client/runner/api/actions.js

```diff
@@ -807,13 +807,10 @@ export function waitFor (event, timeout) {
 }
 
 export function navigateTo (url) {
-    var NAVIGATION_DELAY = 1000;
-
     stepIterator.asyncAction(function (iteratorCallback) {
         hammerhead.navigateTo(url);
 
-        //NOTE: give browser some time to navigate
-        nativeMethods.setTimeout.call(window, iteratorCallback, NAVIGATION_DELAY);
+        iteratorCallback();
```

Comment:
Sure we don't need the `NAVIGATION_DELAY` in the new api. But should we remove it from the legacy api? 
@inikulin as I remember you are the one who implemented the `navigateTo` action in legacy api. Do you have thoughts why we were need this delay?